jobs/tcp_router/templates/killer

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -eo pipefail
+
+PID=$(/var/vcap/jobs/bpm/bin/bpm pid tcp_router -p haproxy)
+
+echo "going to kill ${PID}"
+kill -USR1 "${PID}"
+echo "killed"
```

Comment:
is this script really a `killer` or just a `config_reloader`?  as someone skimming things from a high-level, I might get the mistaken impression that this script terminates the process.

Reply:
agreed - name change would clarify its intent.